### DISCUSSION TITLE
Projects - SubTasks - Group When Displayed on Project Board

### DISCUSSION THREAD

**Discussion by mitchelsellers** [community/community#6413] [Projects and Issues]
When I have an issue that has for example 5 tasks in the issue, and each of these tasks are converted to issues, it would be helpful to have a view on the Projects board that summarizes the progress and groups the issues that are within this parent issue, rather than showing all of the sub-tasks at the same level

**Comment by mitchelsellers** [comment]
Looking at this further, it would be an alternative to NOT include the sub-tasks on the project board at all, but show the same "x of y" counts on the project board so that we can understand that there is more to that item than just one little thing.

**Comment by HX-Hirahi** [comment]
> When I have an issue that has for example 5 tasks in the issue, and each of these tasks are converted to issues, it would be helpful to have a view on the Projects board that summarizes the progress and groups the issues that are within this parent issue, rather than showing all of the sub-tasks at the same level

Yes! We have the same need: It would be fantastic if we can visually group by any of the fields in the board. And if we could move the tasks into the next column and that we keep this visual grouping, maintaining the tasks in the same group row.

Something like this:
![VisualGroupingInBoard](https://user-images.githubusercontent.com/49900203/155309633-421e0021-549b-48f6-a6dc-9592d09c1542.PNG)

**Comment by HX-Hirahi** [comment]
I have just seen there is a similar proposal here: https://github.com/github/feedback/discussions/5654
let's also vote that :-)

**Comment by aptiko** [comment]
#5654 is  a different thing. This isn't about swim lanes. It's about somehow showing in the board that an issue is a subtask of another issue. ZenHub shows the "parent" inside each issue card; you can also click on a button on the parent card to show only itself and its children on the board. Other kanban systems show subtasks as smaller cards inside a larger card.

**Comment by rileybroughten** [comment]
We've recently released the ability to slice your project views by a field, check out the changelog announcement [here](https://github.blog/changelog/2023-08-10-github-issues-projects-august-10th-update/#slice-by)!

If you slice by the `Tracked by` field, you can drill into each issue and view the issues tracked within it on the view on the right. Curious to see how this helps your use case and anything else you're looking to accomplish!

<img width="1436" alt="Screenshot 2023-09-07 at 12 17 30 PM" src="https://github.com/community/community/assets/101840513/c604c750-62a0-4645-a9d7-b0b55a980079">

**Reply by mheptinstall** [reply]
I see the [docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-tracks-and-tracked-by-fields) on this and it is in beta but who is this currently available for? This field is not available in any new/existing projects for me.

Edit: After reading the docs again it sounds like this field is only available when using the new task lists that is in private beta.

**Comment by andrewtheis** [comment]
Any updates on this feature? I noticed what looks like sub-tasks in this screenshot on the docs:

https://docs.github.com/assets/cb-97546/mw-1440/images/help/projects-v2/example-roadmap.webp

But doesn't seem possible without access to this feature?

**Reply by turowicz** [reply]
Group by Tracked By

**Comment by pbower** [comment]
I can't believe there are no subtasks.

**Comment by zhukovgeorge** [comment]
Who knows if there is any news about this feature or future plans for it?

**Comment by pbower** [comment]
Literally ditched Github boards and went to JIRA because of the lack of this basic feature.

**Comment by Voxar** [comment]
Also dissing GH as alternative to Jira for now due to this

**Comment by ebndev** [comment]
Hi all!

We announced the Public Preview of [Sub-Issues](https://github.com/orgs/community/discussions/148714) earlier this year. If you have additional comments, I encourage you to add them in that thread so the product team can track your feedback there. 😄

**Comment by Christian-onsai** [comment]
+1

**Comment by HaydenReeve** [comment]
+1

This is a really important feature for our company
